package.json

```diff
@@ -48,7 +48,7 @@
     "@nextcloud/moment": "^1.3.1",
     "@nextcloud/paths": "^2.1.0",
     "@nextcloud/router": "^3.0.0",
-    "@nextcloud/vue": "^8.10.0",
+    "@nextcloud/vue": "^8.11.0",
```

Comment:
i will upgrade the vue in a separate pr, but leaving this here as it is easy to test without linking with nc/vue. It will be removed after reviewing